regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions.groovy

```diff
@@ -70,4 +70,8 @@ suite("test_array_functions") {
     qt_select "SELECT k1, array_popback(k5) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_popback(k6) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_popback(k7) from ${tableName} ORDER BY k1"
+    qt_select "SELECT k1, array_with_constant(3, k1) from ${tableName} ORDER BY k1"
+    qt_select "SELECT k1, array_with_constant(10, null) from ${tableName} ORDER BY k1"
```

Comment:
Why the result is null here?

Reply:
fixed, use [ALWAYS_NOT_NULLABLE] to make the return type more precise